SECURITY.md

```diff
@@ -0,0 +1,50 @@
+# Vulnerability Reporting Process
+
+Security is the number one priority for cert-manager. If you think you've found a
+security vulnerability in cert-manager, you're in the right place.
+
+Our reporting procedure is a work-in-progress, and will evolve over time. We
+welcome advice, feedback and pull requests for improving our security
+reporting processes.
+
+## Covered Repositories and Issues
+
+When we say "a security vulnerability in cert-manager" we mean a security issue
+in any repository under the [cert-manger GitHub organization](https://github.com/cert-manager/)
+or in the main [cert-manager repo](https://github.com/jetstack/cert-manager).
+
+This reporting process is intended only for security issues in the cert-manager
+project itself, and doesn't apply to applications _using_ cert-manager or to
+issues which do not affect security.
+
+Broadly speaking, if the issue cannot be fixed by a change to one of the covered
+repositories above, then it might not be appropriate to use this reporting
+mechanism and a GitHub issue in the appropriate repo or a question in Slack
+might be a better choice.
+
+All that said, **if you're unsure** please reach out using this process before
+raising your issue through another channel. We'd rather err on the side of
+caution!
+
+## Reporting Process
+
+1. Describe the issue in English, ideally with some example configuration or
+   code which allows the issue to be reproduced. Explain why you believe this
+   to be a security issue in cert-manager, if that's not obvious.
+2. Put that information into an email. Use a title starting with "SECURITY:"
+3. Send the email to [`cert-manager-maintainers@jetstack.io`](mailto:cert-manager-maintainers@jetstack.io?subject=SECURITY:%20REPLACEME)
```

Comment:
Would it make sense to add here what is the visibility of this/indicate who is in this group- in case the person reporting is interested to know who will read their report?

Reply:
Agreed, it would be nice to let people know who is going to read this mailing list

Reply:
Yeah that's a great idea :+1:

Reply:
It doesn't seem possible to make the owners of a google group visible externally without allowing people to join the group, which seems like it presents a risk of someone seeing something they shouldn't.

Maybe longer term it would be good to have a script which alerts if the membership of the group doesn't match, say, `SECURITY_CONTACTS` which could be created (and publicly visible) in this repo.